design/sketch.webidl

```diff
@@ -527,9 +530,11 @@ dictionary WebGPUDeviceDescriptor {
 };
 
 interface WebGPU {
-    static WebGPUExtensions getExtensions();
-    static WebGPUFeatures getFeatures();
-    static WebGPULimits getLimits();
+    static WebGPU instance();
+
+    WebGPUExtensions getExtensions();
+    WebGPUFeatures getFeatures();
+    WebGPULimits getLimits();
 
```

Comment:
Thanks for taking a look, Rafael!

The story about discovering hardware capabilities has not been explored by the group in depth yet. Perhaps, we don't even need `getFeatures()` and `getLimits()` (since those are too detailed and hinder both portability and finger-printing) and instead have tiered fields of the API or feature levels like DX12.

As for now, I'm fine with moving the methods into the `WebGPUDevice`.